Source/WebInspectorUI/UserInterface/Views/SingleSidebar.css

```diff
@@ -47,6 +47,16 @@
     border-inline-end: 1px solid var(--border-color);
 }
 
-.single-sidebar.trailing {
+body:not(.narrow) .single-sidebar.trailing {
     border-inline-start: 1px solid var(--border-color);
 }
+
+body.narrow .sidebar.trailing .single-sidebar {
+    width: 100% !important;
```

Comment:
is this actually needed anymore now that we're setting `this.element.style.width = "";`?

Reply:
That only applies to the parent sidebars, while this is for the sub-sidebars (`SingleSidebar`). 
To be clear, in narrow mode, the height is on the parent sidebar only, and outside it, the widths are on each sub-sidebar, but *not* the parent sidebar.

I've changed it to use flex-grow though, which should obviate the need for `!important` and be more workable should the functionality to have multiple sidebars on the bottom be incorporated.